app/controllers/application_controller.rb

```diff
@@ -63,4 +63,12 @@ def configure_for_accept_invitation
                                                password_confirmation
                                                invitation_token]
   end
+
+  def after_sign_in_path_for(resource)
+    if resource.respond_to?(:pwned?) && resource.pwned?
+      cookies[:pwned] = { value: true, expires: Time.now + 10.minutes }
```

Comment:
Since `pwned` status is only called in the resource during an `after_sign_in_path_for` called, I can't use `user.pwned?` in the `_update_password_modal.html.erb` view as well since that state can no longer be tested and will just return `nil` because the password is only checked with haveIBeenpwned upon login for security reasons. 

I stored this as a temporary cookie to check it right after the user login to make the change password notification pop up if their password was compromised. 

If anyone have a better idea or suggestion, let me know :)